src/java.base/share/classes/jdk/internal/util/jar/JarIndex.java

```diff
@@ -261,7 +261,7 @@ public void write(OutputStream out) throws IOException {
                 /* print out the jar file name */
                 String jar = jarFiles[i];
                 bw.write(jar + "\n");
-                LinkedList<String> jarlist = jarMap.get(jar);
+                ArrayList<String> jarlist = jarMap.get(jar);
```

Comment:
Here again, jarList could probably be declared as `List<String>`

Reply:
Done